ang/crmMailingAB/EditCtrl/report.html

```diff
@@ -70,7 +70,6 @@
         <a
           class="crm-hover-button action-item"
           ng-href="{{statUrl(am.mailing, statType, 'search')}}"
-          ng-if="checkPerm('view all contacts')"
```

Comment:
You might want to check permission for either 'view all contacts' OR 'edit all contacts' for the search to populate the results instead of removing the permission itself. If you have neither of the above - it wouldn't suppress the link yet erroneously show no results due to lack of permission.